<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QDialog" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>715</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1070</width>
    <height>715</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1070</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(236, 240, 241);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2001</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color:rgb(255,0,54);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border-image: url(:/Final_Logo.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="display_username">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font: 18pt &quot;Ubuntu&quot;;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>551</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>2001</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(50,50,50);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/home.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>16</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/search.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cloud">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
	border-image: url(:/cloud-computing.png);
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="notification">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>16</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/notification.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="user">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>14</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/man-user.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="next_pic">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>450</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_pic">
   <property name="geometry">
    <rect>
     <x>988</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>display_username</zorder>
  <zorder>display</zorder>
  <zorder>next_pic</zorder>
  <zorder>prev_pic</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
